Dockerfile

```diff
@@ -8,28 +8,44 @@ RUN apt-get update \
   # psycopg2 dependencies
   && apt-get install -y libpq-dev \
   # Translations dependencies
-  && apt-get install -y gettext \
+  && apt-get install -y gettext wget curl gnupg2 \
+  # Html to PDF Converter Requirements
+  # && apt-get install -y  chromium-browser \
   # cleaning up unused files
   && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
   && rm -rf /var/lib/apt/lists/*
 
+# Install x11vnc.
+# RUN apt-get install -y x11vnc
+# Install xvfb.
+# RUN apt-get install -y xvfb
+# # Install fluxbox.
+# RUN apt-get install -y fluxbox
+# # Install wget.
+# RUN apt-get install -y wget
+# # Install wmctrl.
+# RUN apt-get install -y wmctrl
+# Set the Chrome repo.
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+  && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list
+# Install Chrome.
+RUN apt-get update && apt-get -y install google-chrome-stable
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5765271)